src/praisonai/praisonai/cli/commands/run.py

```
@@ -252,6 +252,81 @@ def _apply_config_defaults(
+        return
+
+    import asyncio
+    import os
+
+    output = get_output_controller()
+    try:
+        from ..features.checkpoints import CheckpointsHandler
+
+        handler = CheckpointsHandler(workspace_dir=os.getcwd())
+        asyncio.run(handler.save(label, allow_empty=False))
+    except Exception as e:  # pragma: no cover - defensive, never block the run
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)